fix: handle JSON.parse exception

unrevised6419 opened the pull request:
With the last version of Strapi, the default value is missing, and trying to parse it will throw an exception.

unrevised6419 commented on README.md:
````diff
@@ -130,6 +130,28 @@ module.exports = [
+```
+
+> **Note**  
+> The `"default": "{}"` seems to be needed in latest versions of Strapi, otherwise Admin will crash at content-type creation
````
Maybe we can fix it in the plugin, and try/catch JSON.parse?

https://github.com/SKLINET/strapi-plugin-video-field/blob/e0f9b401496dea8630e18277bb27bce00d45b9d7/admin/src/components/VideoField/VideoInput/index.js#L33

mateusk replied:
Thanks! This custom field was causing errors in my Strapi instance and this solved it.

unrevised6419 committed:
fix: handle JSON.parse exception

Last version of Strapi makes this fail
If not handling this exception, you need to add an empty object as default value

unrevised6419 commented:
@Sklico can you please take a look at this?

Sklico commented:
Ho @iamandrewluca, sorry, I cannot merge it, that part where you overwrite value of "const" is wrong. We've fixed this error in 1.0.5, so please use this version, it should be fine now. Anyway thanks for finding a problem.

unrevised6419 commented:
@Sklico Thanks! Glad to hear it was fixed!
